### ISSUE TITLE
Panel Sizing performs badly in iframe on Windows

### ISSUE THREAD

**Issue by lbwexler** [xh/hoist-react#2918]
To see this, run our portfolio app in the preview iFrame on windows.  Trying to size the vertical splitter in the detail panel, you quickly get in to a bad state where one side is just permanently maximized.

+ Not reproducible when running the app outside the iFrame
+ Not reproducible when running the app in the iFrame, but on the Mac.  !?

John Haynes and I both encountered this/verified it.

**Comment by amcclain** [comment]
Possible result of https://github.com/xh/hoist-react/pull/2795

**Comment by lbwexler** [comment]
Jakub -- happy to show you this in action.

**Comment by amcclain** [comment]
@lbwexler note that Jakub isn't setup for development on Windows, so this would entail him creating a working dev environment on his home desktop Windows machine (vs. XH laptop).

We should discuss if we need to prioritize this for now given that extra overhead (plus general uncertainty around the ticket) - if it's a good item for Jakub to take on now while he's actively developing / getting up to speed on two client projects.

**Comment by lbwexler** [comment]
Don setup a beautiful test app  for me on a windows machine, and I was able to fix in a jiffy.  Thanks Don!

The clientX and clientY are actually offset differently in iFrames in the two O.S.

 It really does not matter to us generally, because we mostly only care about relative values (startDrag vs. current).  

We were doing an unnecessary check against zero for a final degenerate event we needed to discard-- it turns out that was totally unnecessary -- the screenX/screenY co-ordinates already uniquely identify the degenerate event.
